src/main/java/com/testfabrik/webmate/javasdk/jobs/JobRunState.java

```diff
@@ -0,0 +1,20 @@
+package com.testfabrik.webmate.javasdk.jobs;
+
+public enum JobRunState {
+    RUNNING,
+    FAILED,
+    SUCCEEDED;
+
+    public static JobRunState fromString(String stateString) {
```

Comment:
I think this could be named better as the given string is not arbitrary but the actual state name from the JobRun api.